### PR TITLE
Fast algorithm

### LINKED ISSUE #887: Implementing Fast algorithm for cornor detection
# Aim

- In this script, we are going to implement the `Fast (Features from Accelerated Segment Test) algorithm` of `OpenCV` to detect the corners from any image.

# Details

- By using this script we can easily detect the corners from any image in very little time. Like corners of any house or any object.

### PR DESCRIPTION
**Pull request by ShubhamGupta577**
# Description

Added the script for implementation of Fast Algorithm of OpenCV.

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

- [x] New feature (non-breaking change which adds functionality)
## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

**Comment by ShubhamGupta577** [comment]
@kaustubhgupta Please review this PR.

File: Fast Algorithm/Fast_Algorithm.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+'''
+  For using this Script you need to install OpenCV in your machine
+'''
+#Importing openCV library
+import cv2 as cv
+
+#Taking path of input from the user
+path=input("Enter the path of uploaded image: ")
```
Comment (kaustubhgupta) [inline Fast Algorithm/Fast_Algorithm.py:8]:
```suggestion
path=input("Enter the path of image: ")
```

File: Fast Algorithm/Fast_Algorithm.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+# Disable nonmaxSuppression
+fast.setNonmaxSuppression(0)
+kp = fast.detect(img,None)
+
+print( "Total Keypoints without nonmaxSuppression: {}".format(len(kp)) )
+final = cv.drawKeypoints(img, kp, None, color=(255,0,0))
+
+#Printing the final output image
+cv.imshow('Final',final)
```
Comment (kaustubhgupta) [inline Fast Algorithm/Fast_Algorithm.py:36]: Save the final image. Name it as: "image_name (detected).jpg" . Also, make sure to save it in the same folder where the script lies. For doing this, add this to the saving image path: `./folder-name/image_name (detected).jpg`

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
Rename folder to Fast Algorithm (Corner Detection). Other changes:

**Comment by ShubhamGupta577** [comment]
@kaustubhgupta, Requested changes are done. Please review this.

**Review by Kushal997-das — APPROVED** [review]
LGTM 💯

File: Fast Algorithm (Corner Detection)/Fast_Algorithm.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+# Disable nonmaxSuppression
+fast.setNonmaxSuppression(0)
+kp = fast.detect(img,None)
+
+print( "Total Keypoints without nonmaxSuppression: {}".format(len(kp)) )
+final = cv.drawKeypoints(img, kp, None, color=(255,0,0))
+
+#Saving the image
+cv.imwrite('./folder-name/image_name (detected).jpg',final)
```
Comment (kaustubhgupta) [inline Fast Algorithm (Corner Detection)/Fast_Algorithm.py:36]: by image name, I mean to say if I passed an image named "abc.jpg", then the output image should be "abc (detected) .jpg" 

Fix this. (Hint: Use f-strings)

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Review by kaustubhgupta — APPROVED** [review]

**Review by antrikshmisri — APPROVED** [review]

**Review by santushtisharma10 — APPROVED** [review]
LGTM

**Review by avinashkranjan — APPROVED** [review]
LGTM..🚩

**Pull request merged**